cmd/hcl2-schema/hcl2-schema.go

```diff
@@ -0,0 +1,313 @@
+package main
```

Comment:
This file was the first version of codegen; it didn't quite work because it was simply textual and some types are cross packaged and we needed to know their fields.